src/test/scala/com/ubirch/webui/services/ApiAuthSpec.scala

```diff
@@ -70,7 +70,7 @@ class ApiAuthSpec extends FeatureSpec with TestBase {
     scenario("bad token -> error") {
       val devicePwd = realmPopulation.getUser("chrisx").get.userResult.is.getPasswordForDevice()
       val token = generateTokenUser(giveMeADeviceHwDeviceId(), devicePwd)
-      get("/simpleDeviceInfo", Map.empty, Map(FeUtils.tokenHeaderName -> (s"bearer $token" + "a"))) {
+      get("/simpleDeviceInfo", Map.empty, Map(FeUtils.tokenHeaderName -> (s"bearer $token" + "aa"))) {
```

Comment:
I don't know why with 1 "a" is taken as a correct token, but "aa" is taken as a wrong token. not sure it's critical.